config/config.json

```diff
@@ -1,11 +1,23 @@
 {
   "production": {
-    "apiBase": "https://api.XXXXX.com/"
+    "apiEndpoints": {
+      "routes": "http://admin.XXXXX.com/wp-json/lean/v2/routes",
+      "static": "http://admin.XXXXX.com/wp-json/lean/v2/static",
+      "adminBar": "http://admin.XXXXX.com/wp-json/lean/v2/admin-bar"
+    }
   },
   "staging": {
-    "apiBase": "http://api.XXXXX.moxie-staging.com/"
+    "apiEndpoints": {
+      "routes": "http://admin.XXXXX.moxie-staging.com/wp-json/lean/v2/routes",
+      "static": "http://admin.XXXXX.moxie-staging.com/wp-json/lean/v2/static",
+      "adminBar": "http://admin.XXXXX.moxie-staging.com/wp-json/lean/v2/admin-bar"
+    }
   },
   "dev": {
-    "apiBase": "http://localhost:3000/"
+    "apiEndpoints": {
```

Comment:
Yes, we're not using json-server anymore. It's left in the project just in case we need it in the future.

Yes, we could do this with the apiBase, it just makes the code a bit more complex in appConfig. Let me look into it.